plugins/woocommerce/src/Blocks/BlockTypes/AddToCartWithOptions.php

```diff
@@ -222,6 +222,19 @@ protected function render( $attributes, $content, $block ) {
 				$template_part_blocks,
 				$hooks_after,
 			);
+
+			ob_start();
+
+			remove_action( 'woocommerce_' . $product_type . '_add_to_cart', 'woocommerce_' . $product_type . '_add_to_cart', 30 );
+			/**
+			 * Trigger the single product add to cart action that prints the markup.
+			 *
+			 * @since 9.7.0
```

Comment:
Shouldn't it say 9.9?

```suggestion
			 * @since 9.9.0
```

Reply:
Ups, yes. Thanks for noticing! I missed it when copy-pasting. :man_facepalming: